### ISSUE TITLE
The app built from source cannot use

### ISSUE THREAD

**Issue by boquanzhou** [BloopAI/bloop#894]
**Describe the bug**
I try to build the app from source(on windows), it succeed. Howvever on the initial screen, when I 
click connect account, it opens up the local computer.

**Expected behavior**
It is expected to open the github connection link.

**To Reproduce**
How can we reproduce the bug? Helpful information could include:
- Details about your system: windows
- The data that you had indexed
- The query that you ran

**Screenshots or output**
If applicable, add screenshots to help illustrate the bug.

**Additional context**

**Comment by boquanzhou** [comment]
Why is this step not required for downloaded version? I still don't know how to operate that

**Comment by boquanzhou** [comment]
![image](https://github.com/BloopAI/bloop/assets/140485934/2f69ca70-1464-4c18-b9c4-ab9f44723bbe)
Is that what you're talking about?

**Comment by rsdy** [comment]
Correct, this is one file you can use, and I think it will make it into the bundle. For the binary version we already set up the configs during the CI process, so you don't need to deal with this.

**Comment by boquanzhou** [comment]
I did this, but after the compilation, I still had the same problem as before.

**Comment by boquanzhou** [comment]
Who can help me solve this problem?

**Comment by rsdy** [comment]
Can you please go through what steps and exact commands you used to build?

**Comment by boquanzhou** [comment]
I just use npm run start-app to start the app, and I follow your advice to put the config under src-tauri

**Comment by boquanzhou** [comment]
It failed
![image](https://github.com/BloopAI/bloop/assets/140485934/431837d8-2b1a-4a4a-8218-3d783d9a6981)

**Comment by ggordonhall** [comment]
You can now build and run bloop from source without providing these credentials. Instructions here: https://github.com/BloopAI/bloop?tab=readme-ov-file#building-from-source
